app/models/course/assessment/answer/scribing_scribble.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+class Course::Assessment::Answer::ScribingScribble < ActiveRecord::Base
+  belongs_to :answer, class_name: Course::Assessment::Answer::Scribing.name, inverse_of: :scribbles
+  belongs_to :user, inverse_of: nil, foreign_key: true
```

Comment:
Technically the creator would be the same as the user, since it is the person writing the scribble? 

Whether or not the teacher can modify/erase the scribble of the student (and vice versa) should be handled either through permissions (backend) or front-end validations.